### ISSUE TITLE
Non-zero timestamp from triggered devices

### ISSUE THREAD

**Issue by mwilliams294** [eudaq/eudaq#592]
When using the code fragment below with data from a triggered device (in our case mimosa26), it is expected that the timestamp from the plane should always be 0 :
```
auto plane = evt->GetPlane(static_cast<size_t>(plane_id));
for (unsigned int i=0; i < plane.HitPixels(); i++){
    if (plane.GetTimestamp(i) == 0){
        % do A %;
    } else {
        % do B %;
    }
```

However, it is observed that this value is 0 for some pixels in a plane but 1 for others, even though all pixels have the same trigger ID assigned to them.

**Comment by lhuth** [comment]
Hi @mwilliams294,
Thanks for pointing that out. I could reproduce it. However, there seems to be no obvious reason why it happens. I'll dig deeper and keep everyone updated here.

**Comment by lhuth** [comment]
I think I found it. See  #593. Could you check-out the branch and see if it is working for you now? 
And I think you wanted to use `plane.GetTimestamp(i)`, or?

**Comment by mwilliams294** [comment]
Hi @lhuth,

#593 seems to have fixed the problem as I am only seeing `0` returned for the same set of mimosa26 data.

Thanks for the fix!

**Comment by simonspa** [comment]
Fixed by #593
